src/aiida/transports/util.py

```diff
@@ -64,6 +64,14 @@ def close(self):
                     break
                 time.sleep(0.2)
 
+        for f in [self.process.stdout, self.process.stderr, self.process.stdin]:
+            if f is None:
+                continue
+            try:
+                f.close()
+            except ValueError:
+                pass
```

Comment:
I don't understand why ignore the `ValueError` specifically?

Reply:
This will be raised iof the file handle was already closed, which can happen

Reply:
I see, thanks.